### ISSUE TITLE
Indexer - Cinemageddon - search from this indexer returns undef error

### ISSUE THREAD

**Issue by Junkbite** [Prowlarr/Prowlarr#175]
**Describe the bug**

seraching from this indexer only causes an error in the search

**To Reproduce**

**Screenshots**

![image](https://user-images.githubusercontent.com/62021344/121276091-ff153c80-c89b-11eb-977a-ca75258e3b63.png)
![image](https://user-images.githubusercontent.com/62021344/121276128-0e948580-c89c-11eb-84de-d744c9a4a4b8.png)

**Platform Information (please complete the following information):**
```
Version
0.1.0.413
Package Version
nightly-86a7011 by hotio
.NET
Yes (5.0.6)
Docker
Yes
DB Migration
7
```

Turn on Trace logs under Settings -> General and wait for the bug to occur again.
**Upload the full log file here (or another site (e.g. pastebin) and link it). Issues will be closed, if they do not include this!**

```
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'retention' defaultValue:'0'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'historycleanupdays' defaultValue:'365'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'rsssyncinterval' defaultValue:'60'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'availabilitydelay' defaultValue:'0'
2021-06-08 20:52:57.0|Trace|EventAggregator|ConfigFileSavedEvent <~ ReconfigureLogging
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'importlistsyncinterval' defaultValue:'240'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'listsynclevel' defaultValue:'disabled'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'importexclusions' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'maximumsize' defaultValue:'0'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'minimumage' defaultValue:'0'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'enablecompleteddownloadhandling' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'preferindexerflags' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'logindexerresponse' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'allowhardcodedsubs' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'whitelistedhardcodedsubs' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'removecompleteddownloads' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'autoredownloadfailed' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'removefaileddownloads' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'downloadclientworkingfolders' defaultValue:'_UNPACK_|_FAILED_'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'checkforfinisheddownloadinterval' defaultValue:'1'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'downloadclienthistorylimit' defaultValue:'30'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'firstdayofweek' defaultValue:'0'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'calendarweekcolumnheader' defaultValue:'ddd M/D'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'shortdateformat' defaultValue:'MMM D YYYY'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'longdateformat' defaultValue:'dddd, MMMM D YYYY'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'timeformat' defaultValue:'h(:mm)a'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'showrelativedates' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'enablecolorimpairedmode' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'movieinfolanguage' defaultValue:'1'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'uilanguage' defaultValue:'1'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'cleanupmetadataimages' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxytype' defaultValue:'Http'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyhostname' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyport' defaultValue:'8080'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyusername' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxypassword' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxybypassfilter' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxybypasslocaladdresses' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'backupfolder' defaultValue:'Backups'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'backupinterval' defaultValue:'7'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'backupretention' defaultValue:'28'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'certificatevalidation' defaultValue:'Enabled'
2021-06-08 20:52:57.0|Trace|EventAggregator|Publishing ConfigSavedEvent
2021-06-08 20:52:57.0|Trace|EventAggregator|ConfigSavedEvent ~> LocalizationService
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|EventAggregator|ConfigSavedEvent <~ LocalizationService
2021-06-08 20:52:57.0|Trace|EventAggregator|Publishing HealthCheckCompleteEvent
2021-06-08 20:52:57.0|Trace|EventAggregator|HealthCheckCompleteEvent -> HealthController
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxytype' defaultValue:'Http'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyhostname' defaultValue:''
2021-06-08 20:52:57.0|Trace|EventAggregator|HealthCheckCompleteEvent <- HealthController
2021-06-08 20:52:57.0|Trace|EventAggregator|HealthCheckCompleteEvent ~> NotificationService
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyport' defaultValue:'8080'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxyusername' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxypassword' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxybypassfilter' defaultValue:''
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'proxybypasslocaladdresses' defaultValue:'True'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'certificatevalidation' defaultValue:'Enabled'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'backupfolder' defaultValue:'Backups'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'backupinterval' defaultValue:'7'
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'backupretention' defaultValue:'28'
2021-06-08 20:52:57.0|Trace|EventAggregator|HealthCheckCompleteEvent <~ NotificationService
2021-06-08 20:52:57.0|Trace|ConfigService|Using default config value for 'historycleanupdays' defaultValue:'365'
2021-06-08 20:52:57.0|Trace|Http|Req: 230 [GET] /api/v1/health (from 192.168.1.64)
2021-06-08 20:52:57.0|Trace|Http|Res: 230 [GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Debug|Api|[GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Trace|Http|Res: 229 [PUT] /api/v1/config/host: 202.Accepted (39 ms)
2021-06-08 20:52:57.0|Debug|Api|[PUT] /api/v1/config/host: 202.Accepted (39 ms)
2021-06-08 20:52:57.0|Trace|Http|Req: 231 [GET] /api/v1/health (from 192.168.1.64)
2021-06-08 20:52:57.0|Trace|Http|Res: 231 [GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Debug|Api|[GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Trace|Http|Req: 232 [GET] /api/v1/health (from 192.168.1.64)
2021-06-08 20:52:57.0|Trace|Http|Res: 232 [GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Debug|Api|[GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Trace|Http|Req: 233 [GET] /api/v1/health (from 192.168.1.64)
2021-06-08 20:52:57.0|Trace|Http|Res: 233 [GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:52:57.0|Debug|Api|[GET] /api/v1/health: 200.OK (0 ms)
2021-06-08 20:53:04.6|Trace|Http|Req: 234 [GET] /api/v1/search?query=test&indexerIds=21 (from 192.168.1.64)
2021-06-08 20:53:04.6|Info|NzbSearchService|Searching indexers: [Cinemageddon] for {Term: test, Offset: 0, Limit: 0, Categories: []}
2021-06-08 20:53:04.6|Trace|ConfigService|Using default config value for 'logindexerresponse' defaultValue:'False'
2021-06-08 20:53:04.6|Debug|Cardigann|Populating config vars
2021-06-08 20:53:04.6|Info|Cardigann|Adding request: https://cinemageddon.net/browse.php?search=test
2021-06-08 20:53:04.6|Debug|Cardigann|Downloading Feed https://cinemageddon.net/browse.php?search=test
2021-06-08 20:53:04.6|Trace|ConfigService|Using default config value for 'logindexerresponse' defaultValue:'False'
2021-06-08 20:53:04.6|Trace|HttpClient|Req: [GET] https://cinemageddon.net/browse.php?search=test
2021-06-08 20:53:04.6|Trace|ConfigService|Using default config value for 'proxyenabled' defaultValue:'False'
2021-06-08 20:53:05.0|Trace|HttpClient|Res: [GET] https://cinemageddon.net/browse.php?search=test: 200.OK (368 ms)
2021-06-08 20:53:05.0|Debug|Cardigann|Parsing
2021-06-08 20:53:05.0|Trace|ConfigService|Using default config value for 'logindexerresponse' defaultValue:'False'
2021-06-08 20:53:05.0|Debug|Cardigann|Got 15 releases
2021-06-08 20:53:05.0|Trace|EventAggregator|Publishing IndexerQueryEvent
2021-06-08 20:53:05.0|Trace|EventAggregator|IndexerQueryEvent -> HistoryService
2021-06-08 20:53:05.0|Trace|EventAggregator|IndexerQueryEvent <- HistoryService
2021-06-08 20:53:05.0|Debug|NzbSearchService|Total of 15 reports were found for {Term: test, Offset: 0, Limit: 0, Categories: []} from 1 indexers
2021-06-08 20:53:05.0|Trace|Http|Res: 234 [GET] /api/v1/search?query=test&indexerIds=21: 200.OK (459 ms)
2021-06-08 20:53:05.0|Debug|Api|[GET] /api/v1/search?query=test&indexerIds=21: 200.OK (459 ms)
2021-06-08 20:53:09.6|Trace|Http|Req: 235 [PUT] /api/v1/config/host (from 192.168.1.64)
2021-06-08 20:53:09.6|Trace|EventAggregator|Publishing ConfigFileSavedEvent
2021-06-08 20:53:09.6|Trace|EventAggregator|ConfigFileSavedEvent ~> ReconfigureLogging
2021-06-08 20:53:09.6|Trace|ConfigService|Using default config value for 'retention' defaultValue:'0'
2021-06-08 20:53:09.6|Trace|EventAggregator|Publishing HealthCheckCompleteEvent
2021-06-08 20:53:09.6|Trace|EventAggregator|HealthCheckCompleteEvent -> HealthController
2021-06-08 20:53:09.6|Trace|ConfigService|Using default config value for 'historycleanupdays' defaultValue:'365'
2021-06-08 20:53:09.6|Trace|ConfigService|Using default config value for 'rsssyncinterval' defaultValue:'60'
2021-06-08 20:53:09.6|Trace|EventAggregator|HealthCheckCompleteEvent <- HealthController
2021-06-08 20:53:09.6|Trace|ConfigService|Using default config value for 'availabilitydelay' defaultValue:'0'
2021-06-08 20:53:09.6|Trace|ConfigService|Using default config value for 'importlistsyncinterval' defaultValue:'240'
2021-06-08 20:53:09.6|Trace|ConfigService|Using default config value for 'listsynclevel' defaultValue:'disabled'
```

trace logs didnt seem to come up with anything but this is the console:
```
instrument.js:109 TypeError: Cannot read property 'filter' of undefined
    at Hw (CategoryLabel.js:6)
    at ra (react-dom.production.min.js:157)
    at Ws (react-dom.production.min.js:267)
    at Mu (react-dom.production.min.js:250)
    at wu (react-dom.production.min.js:250)
    at ku (react-dom.production.min.js:250)
    at hu (react-dom.production.min.js:243)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:18)
    at Bo (react-dom.production.min.js:122)
    at Uo (react-dom.production.min.js:123)
    at Vo (react-dom.production.min.js:122)
    at pu (react-dom.production.min.js:244)
    at Object.notify (Subscription.js:19)
    at e.t.notifyNestedSubs (Subscription.js:90)
    at e.t.handleChangeWrapper (Subscription.js:95)
    at Object.y [as dispatch] (redux.js:296)
    at dispatch (<anonymous>:3856:17)
    at index.js:11
    at middleware.js:26
    at createSentryMiddleware.js:54
    at dispatch (redux.js:659)
    at Object.<anonymous> (createFetchHandler.js:21)
    at f (callbacks.js:84)
    at fireWith (callbacks.js:214)
    at f (deferred.js:183)
    at h (deferred.js:192)
    at r (helpers.js:72)
```

**Comment by Junkbite** [comment]
i realize this is from not selecting any categories but even with categories selected, nothing seems to come back

**Comment by Qstick** [comment]
Can you provide API response from Chrome network tab on search endpoint from this search?

**Comment by Qstick** [comment]
Should be fixed in 9675171af

**Comment by Junkbite** [comment]
confirmed fix. thanks!
